src/audio-source.ts

```diff
@@ -27,16 +71,20 @@ export class AudioSource extends Component {
         engine.registerComponent(AudioListener);
     }
 
-    /**
-     * Maximum volume a source can have. From 0 to 1 (0% to 100%).
-     */
-    @property.float(1.0)
-    maxVolume!: number;
```

Comment:
I think you should probably provide a setter and getter that will set/get `volume` when you set maxVolume, otherwise you're breaking the API.

Mark those as `@deprecated Use {@link #volume} instead.`.